local_planner/src/nodes/star_planner.cpp

```diff
@@ -144,7 +100,11 @@ void StarPlanner::buildLookAheadTree() {
           tree_.back().last_e_ = p_pol.e;
           tree_.back().last_z_ = p_pol.z;
           float h = treeHeuristicFunction(tree_.size() - 1);
-          float c = treeCostFunction(tree_.size() - 1);
+          float distance_cost = 0.f;
+          float c = 0.f;
+          Eigen::Vector2i idx_ppol = polarToHistogramIndex(p_pol, ALPHA_RES);
+          float obstacle_distance = histogram.get_dist(idx_ppol.x(), idx_ppol.y());
+          costFunction(p_pol.e, p_pol.z, obstacle_distance, goal_, node_location, cost_params_, distance_cost, c);
           tree_.back().heuristic_ = h;
           tree_.back().total_cost_ = tree_[origin].total_cost_ - tree_[origin].heuristic_ + c + h;
```

Comment:
I'd prefer to see the `*0.1` scaling values over here, where we actually assemble the costs together. At some point we can make it something like a dot product between weights and a costs vectors.

Reply:
I took it out altogether